test/test_verify.py

```diff
@@ -47,7 +47,7 @@ def __init__(self, kafkaAddress, schemaRegistryAddress, kafkaConnectAddress, cre
         self.httpHeader = {'Content-type': 'application/json', 'Accept': 'application/json'}
 
         self.SEND_INTERVAL = 0.01  # send a record every 10 ms
-        self.VERIFY_INTERVAL = 60  # verify every 60 secs
+        self.VERIFY_INTERVAL = 10  # verify every 60 secs
```

Comment:
I think 10sec is too short and might cause more flakiness, I suggest we keep at 60

Reply:
ahh yes, this was intended for testing but thanks for the catch!

Reply:
reverted.